src/dune_lang/action.ml

```diff
@@ -350,6 +351,12 @@ let cstrs_dune_file t =
     , Syntax.since Stanza.syntax (2, 7)
       >>> let+ script = sw in
           Cram script )
+  ; ( "needed_deps"
+    , let* xs = repeat1 sw in
+    (if List.length xs > 1
+     then Syntax.since ~what:"Passing several arguments to 'needed_deps'" Stanza.syntax (3, 4)
+     else return ())
```

Comment:
What is this for?

Reply:
It should not be here. This was used in the `cat` parser to alert passing several arguments, since then the execution of `cat` depends on the version of dune.